packages/react/src/components/Button/BaseButton.tsx

```diff
@@ -563,7 +563,7 @@ export class BaseButton extends React.Component<IBaseButtonProps, IBaseButtonSta
     }
   };
 
-  private _onToggleMenu = (shouldFocusOnContainer: boolean): void => {
+  private _onToggleMenu = (shouldFocusOnContainer: boolean = false): void => {
```

Comment:
Changed it so that the `shouldFocusOnContainer` is by default false, unless it is set specifically to true. 

`shouldFocusOnContainer` is also a prop for the Contextual Menu and used a lot for Button component. Removing this option completely would cause API changes, so I was not sure if that is something we would want to do?

Reply:
I don't think this change is necessary, since it already defaults to undefined, which will have the same result as `false`. If you do default it to `false` explicitly, the docs in `Button.types` should be changed, since they currently specify that it defaults to `undefined`.

Reply:
Yeah that makes sense, I removed the defaulting to false and kept the initial `undefined` as not to have a change when the result is the same. Thank you